src/main/java/de/dhbw/karlsruhe/models/Grammar.java

```diff
@@ -1,12 +1,22 @@
 package de.dhbw.karlsruhe.models;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Grammar {
 
   private String[] terminals;
   private String[] nonTerminals;
   private String[] productions;
```

Comment:
Achtung: Datentyp hat sich in der Branch von @whole01 geändert.

Reply:
Wird denke ich gelöst, wenn alles in main gemerged ist.